cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/Capacity.java

```diff
@@ -11,36 +11,100 @@
 import io.strimzi.api.kafka.model.storage.PersistentClaimStorage;
 import io.strimzi.api.kafka.model.storage.SingleVolumeStorage;
 import io.strimzi.api.kafka.model.storage.Storage;
+import io.strimzi.operator.cluster.model.AbstractModel;
+import io.strimzi.operator.cluster.model.VolumeUtils;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
 
 import java.util.List;
 
 import static io.strimzi.operator.cluster.model.StorageUtils.parseMemory;
 
 public class Capacity {
-    public static final double DEFAULT_BROKER_DISK_MIB_CAPACITY = 100_000;  // in MiB
-    public static final int DEFAULT_BROKER_CPU_UTILIZATION_CAPACITY = 100;  // as a percentage (0-100)
-    public static final double DEFAULT_BROKER_INBOUND_NETWORK_KIB_PER_SECOND_CAPACITY = 10_000;  // in KiB/s
-    public static final double DEFAULT_BROKER_OUTBOUND_NETWORK_KIB_PER_SECOND_CAPACITY = 10_000;  // in KiB/s
+    // CC allows specifying a generic "default" broker entry in the capacity configuration to apply to all brokers without a specific broker entry.
+    // CC designates the id of this default broker entry as "-1".
+    // When using a non-JBOD disk configuration, we use a "default" broker entry to configure all brokers
+    private static final int DEFAULT_BROKER_ID = -1;
+    private static final String DEFAULT_BROKER_DOC = "This is the default capacity. Capacity unit used for disk is in MB, cpu is in percentage, network throughput is in KB.";
+
+    private static final double DEFAULT_BROKER_DISK_MIB_CAPACITY = 100_000;  // in MiB
```

Comment:
I find having the unit at the end of names slightly more readable, for example `DEFAULT_BROKER_DISK_CAPACITY_MIB`. WDYT?

Reply:
How about `DEFAULT_BROKER_DISK_CAPACITY_IN_MIB`?

Reply:
I don't think we necessarily need `IN` but I think it's ok so I leave it up to you if you want to keep it.